wordle-web/tests/game.test.ts

```diff
@@ -0,0 +1,28 @@
+// @vitest-environment nuxt
+import { expect, test } from "vitest";
+import { Game } from "~/scripts/game";
+import { LetterState } from "~/scripts/letter";
+
+test("game", () => {
+  // create game and check if it's created
+  const game = new Game("autos");
+  expect(game.wordToGuess.length).toBe(5);
+});
+
+test("guess-word", () => {
```

Comment:
A great example of how to write tests for game logic. I would recommend following some of the established guidelines for test names, as it makes it more clear what is being tested without reading the code.

[Here is a decent article](https://dzone.com/articles/7-popular-unit-test-naming) on common test naming conventions 

Further, one of the goals of unit testing is that the tests are as atomic as possible. Guessing an incorrect word and guessing a correct word might be different enough to split into two tests, something like "incorrect-guess-has-letters-in-misplaced-and-wrong-state" and "correct-guess-has-all-letters-in-correct-state".

Just something to consider!